spectrocloud/resource_macros.go

```diff
@@ -28,17 +28,20 @@ func resourceMacro() *schema.Resource {
 		SchemaVersion: 2,
 		Schema: map[string]*schema.Schema{
 			"name": {
-				Type:     schema.TypeString,
-				Required: true,
-				ForceNew: true,
+				Type:        schema.TypeString,
+				Required:    true,
+				ForceNew:    true,
+				Description: "The name of the macro.",
```

Comment:
I think we should call this environment variable instead of macros. Macros is not an intuitive name and from a consumption side this really is an environment variable.